pallets/ema-oracle/src/lib.rs

```diff
@@ -88,12 +92,11 @@ pub use types::*;
 #[allow(clippy::all)]
 pub mod weights;
 pub use weights::WeightInfo;
-
-mod benchmarking;
```

Comment:
why need to remove this from here?

Reply:
had a lot of challenge with benchmarking, so the easiest was to replace it to runtime.